testing/CMakeLists.txt

```diff
@@ -17,6 +17,9 @@ HPCC_ADD_SUBDIRECTORY (unittests)
 HPCC_ADD_SUBDIRECTORY (regress)
 install( DIRECTORY regress DESTINATION "./testing" COMPONENT Runtime
          USE_SOURCE_PERMISSIONS
-         PATTERN regress/ecl EXCLUDE
-         PATTERN regress/environment.xml.in EXCLUDE
-         PATTERN regress/CMakeLists.txt EXCLUDE )
+         FILES_MATCHING
+         PATTERN "regress/ecl-test"
+         PATTERN "regress/ecl-test.json"
+         PATTERN "regress/environment.xml"
+         PATTERN "regress/README.rst"
+         PATTERN regress/ecl EXCLUDE )
```

Comment:
I don't. If we are including the suite engine then it makes sense to also include the ecl and key files. Unless @ghalliday  knows differently.